.github/workflows/build.yaml

```diff
@@ -138,7 +138,7 @@ jobs:
 
       - name: Install Linux build dependencies
         run: |
-          sudo apt update; sudo apt install -y libdw-dev swig libpython3-dev libsasl2-dev libjsoncpp-dev libwebsockets-dev libnghttp2-dev ccache ninja-build pixz libbenchmark-dev
+          sudo apt update; sudo apt install -y libdw-dev swig libpython3-dev libsasl2-dev libjsoncpp-dev libwebsockets-dev libnghttp2-dev ccache ninja-build pixz libbenchmark-dev nginx
```

Comment:
so, it's not going to be a testcontainer after all? all things considered, I probably agree with not using test containers for this; it would add extra complications and there is not all that much benefit, since nginx is easy to install on Linux

Reply:
I looked into doing container nginx but this approach seemed like the easiest way to get nginx into our CI system, so I went with it.